### DISCUSSION TITLE
penetration-testing/cheatsheets/nmap-cheatsheet/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#92] [General]
# penetration-testing/cheatsheets/nmap-cheatsheet/

Nmap CheatSheet - In this CheatSheet, you will find a series of practical example commands for running Nmap and getting the most of this powerful tool. Full documentation for the nmap flags

https://3os.org/penetration-testing/cheatsheets/nmap-cheatsheet/

**Comment by lostboyz404** [comment]
i just started to learn, or rather try to learn hacking. I got nmap and i do not understand the command lines i gotta put in. i can get some results but do not possess enough knowledge in the subject. please any pointers would help.thx
